pulsar-broker/src/main/java/org/apache/pulsar/broker/service/ServerCnx.java

```diff
@@ -1995,12 +1995,9 @@ protected void handleEndTxnOnSubscription(CommandEndTxnOnSubscription command) {
 
                 Subscription subscription = optionalTopic.get().getSubscription(subName);
                 if (subscription == null) {
-                    log.error("Topic {} subscription {} is not exist.", optionalTopic.get().getName(), subName);
-                    ctx.writeAndFlush(Commands.newEndTxnOnSubscriptionResponse(
-                            requestId, txnidLeastBits, txnidMostBits,
-                            ServerError.ServiceNotReady,
-                            "Topic " + optionalTopic.get().getName()
-                                    + " subscription " + subName + " is not exist."));
+                    log.warn("Topic {} subscription {} is not exist.", optionalTopic.get().getName(), subName);
```

Comment:
we can use this opportunity to fix the typo and use this message: `Topic {} subscription {} does not exist`

Reply:
ok